packages/fields-document/src/DocumentEditor/Toolbar.tsx

```diff
@@ -188,7 +188,7 @@ const ToolbarContainer = ({ children }: { children: ReactNode }) => {
         paddingTop: spacing.small,
         position: 'sticky',
         top: 0,
-        zIndex: 2,
+        zIndex: 30,
```

Comment:
Any particular reason to use `30`?

Reply:
The zIndex for the sticky bar is 20, so I thought that 30 would be better for the toolbar.